Makefile

```diff
@@ -52,6 +52,10 @@ dataset-load-test:
 	@echo "--- 🚀 Running dataset load test ---"
 	pytest -m test_datasets
 
+dataset-load-test-pr:
+	@echo "--- 🚀 Running dataset load test for PR ---"
+	eval "$$(python -m scripts.extract_datasets $(BASE_BRANCH))" && pytest -m test_datasets
```

Comment:
I would probably refactor to not use tests (`mteb.get_task(...).load_data()` in script)

Reply:
Maybe see [this comment](https://github.com/embeddings-benchmark/mteb/pull/3103#discussion_r2314445569)? I think there's some misunderstanding here over what the test is.